packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -185,6 +185,8 @@ class IOSDevice extends Device {
       } on IOSDeviceNotFoundError catch (error) {
         // Unable to find device with given udid. Possibly a network device.
         printTrace('Error getting attached iOS device: $error');
+      } on IOSDeviceNotTrustedError catch (error) {
+        printTrace('Error getting attached iOS device information: $error');
```

Comment:
@zanderso would we want to send an analytics event here so that if we see a spike in untrusted events, we know something's gone wrong?

Reply:
Yes, that would be a good signal to generate. Maybe something like `UsageEvent('device', 'ios-trust-failure').send();`